whole-body-controller-core/robot-arm-one/build.gradle.kts

```diff
@@ -17,5 +17,7 @@ mainDependencies {
    api("us.ihmc:euclid:0.19.1")
    api("us.ihmc:simulation-construction-set-tools:0.14.0-230207")
```

Comment:
See if that dependency is actually necessary

Reply:
Removed